edit-post/store/middlewares.js

```diff
@@ -17,7 +18,7 @@ import { mobileMiddleware } from '../utils/mobile';
  */
 function applyMiddlewares( store ) {
 	const middlewares = [
-		mobileMiddleware,
```

Comment:
I guess this just never existed and did nothing?

(Note: Will conflict with #5206)

Reply:
I think it was removed in the plugin API sidebar but the files was kept